modules/javafx.base/src/test/java/test/javafx/collections/FXCollectionsTest.java

```diff
@@ -42,6 +42,47 @@
 
 public class FXCollectionsTest {
 
+    @Test
+    public void testCreateObservableArrayListFromArray() {
+        ObservableList<String> observableList = FXCollections.observableArrayList("1", "2", "3");
```

Comment:
I would add a `null` element (like `"1", "2", null`) to test that the method accepts `null`s, especially after I fell for it myself.

Reply:
Good idea, will do.